doc/nrf/ug_nrf5340.rst

```diff
@@ -81,6 +81,9 @@ Trusted Firmware-M (TF-M) support
 ---------------------------------
 
 You can use Trusted Firmware-M (TF-M) as an alternative to :ref:`secure_partition_manager` for running an application from the non-secure area of the memory.
+Support for TF-M in |NCS| is currently experimental, however it is more robust and complete software offering which will be the default option in the future.
```

Comment:
I am not sure about this statement, leaving it here to raise a discussion.

From previous discussions, we should encourage the use of TF-M in applications, however it's a bit hard to do this if we also say that support is experimental. I also don't know if we should make future statements here. @ioannisg  ?

Reply:
For me this is fine as is - setting this as EXPERIMENTAL reflects well the current status. 

>  which will be the default option in the future.

This is also fine, I am just not sure what the right way to signal it would be.
@carlescufi ^^

Reply:
@carlescufi any opinion?

Reply:
"Support for TF-M in |NCS| is currently experimental.  TF-M is a framework which will be extended for new functions and use-cases beyond the scope of SPM."

Reply:
@b-gent ^^

Reply:
Thanks, looks good!